src/page/bank-payment.page.ts

```diff
@@ -4,7 +4,7 @@ export class BankPaymentPage {
   private confirmButton: ElementFinder;
 
   constructor () {
-    this.confirmButton = $('#cart_navigation > button > span');
+    this.confirmButton = $('#cart_navigation > button[type="submit"]');
```

Comment:
could you try to use a different attribute than type? for example, name attribute, will not be susceptible to changes and give us more context about what we are looking for

Reply:
Done